frenai/chain.json

```diff
@@ -0,0 +1,110 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "frenai",
+  "status": "live",
+  "website": "https://fren.ai",
+  "network_type": "mainnet",
+  "pretty_name": "fren.ai",
+  "chain_id": "fren-1",
+  "bech32_prefix": "fren",
+  "daemon_name": "frend",
+  "node_home": "$HOME/.frenai",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ufren",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.00025,
+        "average_gas_price": 0.0025,
+        "high_gas_price": 0.025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ufren"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/frenailabs/frenai/",
+    "recommended_version": "v0.1.0",
+    "compatible_versions": ["v0.1.0"],
+    "binaries": {
+      "linux/amd64": "https://github.com/frenailabs/releases/download/v0.1.0/frenaid-v0.1.0-linux-amd64.tar.gz",
+      "linux/arm64": "https://github.com/frenailabs/releases/download/v0.1.0/frenaid-v0.1.0-linux-arm64.tar.gz"
+    },
+    "cosmos_sdk_version": "v0.50.6",
+    "ibc_go_version": "v8.0.0",
```

Comment:
Can you please add the cosmos_sdk_verions and the ibc_go_version into the versions array as well?

Reply:
I've added the `cosmos_sdk_version` and `ibc_go_version` to the `versions` array as requested. Thank you for the feedback.

Reply:
Sorry, and `consensus`, too, please. (should include all the codebase data that's not git_repo and genesis)